### ISSUE TITLE
Form: `action` invariant rejects on Bluebird promise instance

### ISSUE THREAD

**Issue by kettanaito** [kettanaito/react-advanced-form#166]
```js
handleSubmit = () => {
  return Bluebird(...);
}

render() {
  <Form action={ handleSubmit }>
}
```

**Comment by kettanaito** [comment]
The safest way of checking if what `action` return is see if its return has `then` and `typeof then === 'function'`.

**Comment by kettanaito** [comment]
Implemented under 69755ed88ba74cb69373e5caffc8360f86c6069f.

**Comment by kettanaito** [comment]
New solution covers the invariant gracefully. Closing.
